raft/src/main/scala/zio/raft/Raft.scala

```diff
@@ -558,3 +563,5 @@ object Raft:
       refStateMachine,
       pendingCommands
     )
+      _ <- raft.run.forkManaged
```

Comment:
@somdoron run it inside the makeManaged.

Reply:
understood, but now the `use` in the calculator app main will exit immediately.